### DISCUSSION TITLE
Tried to access UI related API:getWallpaperColors from a non-UI Context:app.lawnchair.LawnchairApp@e8301a3

### DISCUSSION THREAD

**Discussion by Abdul1039** [LawnchairLauncher/lawnchair#4569] [Q&A]
i tried to make default launcher in android 14 but it give me this error on the first boot or factory reset.

**Comment by SuperDragonXD** [comment]
Hello. I know that this is a bit late, but this discussion has been inactive for a while. If you are still experiencing this issue in Lawnchair 15, please create a new *issue* at this repo. Closing this for now.
